json/term.json

```diff
@@ -0,0 +1 @@
": "theme", "name": {"nl": "Fitness, gymnastiek, dans en vechtsport", "fr": "Fitness, gymnastique, danse et arts martiaux", "de": "Fitness, Tanz und Kampfkunst", "en": "Fitness, gymnastics, dance and martial arts"}, "scope": ["events"]}, {"id": "1.58.8.0.0", "domain": "theme", "name": {"nl": "Lucht en motorsport", "fr": "Sports aero et moteurs", "de": "Luft-und Motorsport", "en": "Air and motorsports"}, "scope": ["events"]}, {"id": "1.51.12.0.0", "domain": "theme", "name": {"nl": "Omnisport en andere", "fr": "Omnisports et autres", "de": "Omnisport und andere", "en": "Other sports"}, "scope": ["events"]}, {"id": "1.51.11.0.0", "domain": "theme", "name": {"nl": "Outdoor en Adventure sport", "fr": "Sports de plein air et d'aventure", "de": "Outdoor- und Abenteuersport", "en": "Outdoor and Adventure sports"}, "scope": ["events"]}, {"id": "1.51.10.0.0", "domain": "theme", "name": {"nl": "Volkssporten", "fr": "Sports folkloriques", "de": "Sport Menschen", "en": "Folklore sports"}, "scope": ["events"]}, {"id": "1.51.3.0.0", "domain": "theme", "name": {"nl": "Zwemmen en watersport", "fr": "Natation et sports nautiques", "de": "Schwimmen und Wassersport", "en": "Swimming and water sports"}, "scope": ["events"]}, {"id": "1.37.1.0.0", "domain": "theme", "name": {"nl": "Gezondheid en zorg", "fr": "Sant\u00e9 et soins", "de": "Gesundheit und Pflege", "en": "Health and care"}, "scope": ["events"]}, {"id": "1.43.0.0.0", "domain": "theme", "name": {"nl": "Interculturele vorming", "fr": "Formation interculturelle", "de": "Interkulturelle Erziehung", "en": "Cross-cultural education"}, "scope": ["events"]}, {"id": "1.64.0.0.0", "domain": "theme", "name": {"nl": "Milieu en natuur", "fr": "Environnement et nature", "de": "Umwelt und Natur", "en": "Environment and nature"}, "scope": ["events"]}, {"id": "1.37.0.0.0", "domain": "theme", "name": {"nl": "Opvoeding", "fr": "\u00c9ducation", "de": "Bildung", "en": "Education"}, "scope": ["events"]}, {"id": "1.61.0.0.0", "domain": "theme", "name": {"nl": "Persoon en relaties", "fr": "Hommes et relations", "de": "Person und Gesellschaft", "en": "People and society"}, "scope": ["events"]}, {"id": "1.37.2.0.0", "domain": "theme", "name": {"nl": "Samenleving", "fr": "Vivre en soci\u00e9t\u00e9", "de": "Gesellschaft", "en": "Society"}, "scope": ["events"]}, {"id": "1.65.0.0.0", "domain": "theme", "name": {"nl": "Voeding", "fr": "Nutrition", "de": "Ern\u00e4hrung", "en": "Nutrition"}, "scope": ["events"]}, {"id": "1.25.0.0.0", "domain": "theme", "name": {"nl": "Wetenschap", "fr": "Science", "de": "Wissenschaft", "en": "Science"}, "scope": ["events"]}, {"id": "1.44.0.0.0", "domain": "theme", "name": {"nl": "Zingeving, filosofie en religie", "fr": "Sense de la vie, philosophie et religion", "de": "Philosophie und Religion", "en": "Philosophy and religion"}, "scope": ["events"]}]}, {"id": "8.70.0.0.0", "domain": "eventtype", "name": {"nl": "Theater", "fr": "Th\u00e9\u00e2tre", "de": "Theater", "en": "Theatre"}, "scope": ["places"]}]
```

Comment:
Hmm I just realized that an array of objects is usually not best practice as an HTTP response body, because it's not extensible later with extra properties.

So we should change it to something like:
```
{
  "terms": [... current output of your script ...]
}
```

That way we can easily add extra properties later if ever needed.

I'll also update the Stoplight docs to reflect it. My bad of only thinking of that now.

Reply:
Having an encapsulating hash, even when your data is really an array, is indeed easier (although technically both are supported).

Reply:
I've made the necessary changes.